flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBMapState.java

```diff
@@ -218,22 +217,20 @@ public UV next() {
 	public void clear() {
 		try {
 			try (RocksIteratorWrapper iterator = RocksDBKeyedStateBackend.getRocksIterator(backend.db, columnFamily);
-				WriteBatch writeBatch = new WriteBatch(128)) {
+				RocksDBWriteBatchWrapper rocksDBWriteBatchWrapper = new RocksDBWriteBatchWrapper(backend.db, backend.getWriteOptions())) {
```

Comment:
As the doc said `RocksDBWriteBatchWrapper` is not thread safe, after reusing, will there be a concurrent problem?